microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md

```diff
@@ -2,26 +2,28 @@
 title: Create an extension tutorial, part 2
 description: Dynamically inserting a NASA picture below the page body tag by using content scripts.
 author: MSEdgeTeam
-ms.author: msedgedevrel
+ms.author: v-jandrew    
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 06/28/2022
+ms.date: 08/05/2022
 ---
 # Create an extension tutorial, part 2
 
-To see the completed extension package source for this part of the tutorial, go to [MicrosoftEdge-Extensions repo > extension-getting-started-part2](https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2).  The source code has been updated from Manifest V2 to Manifest V3.
+To see the completed extension package source for this part of the tutorial, go to [MicrosoftEdge-Extensions repo > extension-getting-started-part2](https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2).  
+
+The source code has been updated from Manifest V2 to Manifest V3.
 
 This tutorial covers the following extension technologies:
-*   Injecting JavaScript libraries into an extension.
+*   Injecting JavaScript libraries into an extension.  
```

Comment:
```suggestion
*   Injecting JavaScript libraries into an extension.
```

Reply:
Two trailing spaces = linebreak in Markdown.  We converted repo to get rid of trailing spaces on every line.  A single trailing space is innocuous, but trying to keep repo streamlined per new convention.